code/game/gamemodes/changeling/modularchangling.dm

```diff
@@ -1,199 +1,102 @@
 
-/datum/power/changeling
-	var/allowduringlesserform = 0
-	var/genomecost = 500000 // Cost for the changling to evolve this power.
-
-/datum/power/changeling/absorb_dna
-	name = "Absorb DNA"
-	desc = "Permits us to syphon the DNA from a human. They become one with us, and we become stronger."
-	genomecost = 0
-	verbpath = /mob/proc/changeling_absorb_dna
-
-/datum/power/changeling/transform
-	name = "Transform"
-	desc = "We take on the apperance and voice of one we have absorbed."
-	genomecost = 0
-	allowduringlesserform = 1
-	verbpath = /mob/proc/changeling_transform
-
-/datum/power/changeling/fakedeath
-	name = "Regenerative Stasis"
-	desc = "We become weakened to a death-like state, where we will rise again from death."
-	helptext = "Can be used before or after death. Duration varies greatly."
+/datum/power/carrion
+	var/genomecost = 500000 // Cost for the carrion to evolve this power.
+	var/organpath //Path to the organ that is getting evolved.
+	var/spiderpath //The path of the spider we spawn.
+
+/datum/power/carrion/flashbang_spider
+	name = "Flashbang spider"
+	desc = "Evolves a spider filled with a strange substance that when activated explodes in a flash of light. Does minor damage to its host."
+	genomecost = 5
+	spiderpath = /obj/item/weapon/implant/carrion_spider/flashbang
+
+/datum/power/carrion/infection_spider
+	name = "Infection spider"
+	desc = "Evolves a miniature spider core spider capable of making more carrions."
+	helptext = "Requires 5 evolution points to produce."
 	genomecost = 0
-	allowduringlesserform = 1
-	verbpath = /mob/proc/changeling_fakedeath
+	spiderpath = /obj/item/weapon/implant/carrion_spider/infection
 
-// Hivemind
-
-/datum/power/changeling/hive_upload
-	name = "Hive Channel"
-	desc = "We can channel a DNA into the airwaves, allowing our fellow changelings to absorb it and transform into it as if they acquired the DNA themselves."
-	helptext = "Allows other changelings to absorb the DNA you channel from the airwaves. Will not help them towards their absorb objectives."
-	genomecost = 0
-	verbpath = /mob/proc/changeling_hiveupload
-
-/datum/power/changeling/hive_download
-	name = "Hive Absorb"
-	desc = "We can absorb a single DNA from the airwaves, allowing us to use more disguises with help from our fellow changelings."
-	helptext = "Allows you to absorb a single DNA and use it. Does not count towards your absorb objective."
-	genomecost = 0
-	verbpath = /mob/proc/changeling_hivedownload
+/datum/power/carrion/healing_spider
+	name = "Healing spider"
+	desc = "Evolves a spider filled with a mixture of medicinal chemicals."
+	genomecost = 4
+	spiderpath = /obj/item/weapon/implant/carrion_spider/healing
 
-/datum/power/changeling/lesser_form
-	name = "Lesser Form"
-	desc = "We debase ourselves and become lesser.  We become a monkey."
+/datum/power/carrion/breeding_spider
+	name = "Breeding spider"
+	desc = "Evolves a spider mother carrying eggs, when in a dead host and active, the eggs will hatch."
```

Comment:
```suggestion
	desc = "Creates a spider carrying eggs, when it will be put inside a dead host and activated, the eggs will give birth to many lesser ones of your kin."
```